firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -280,3 +304,156 @@ uint8_t TCMDEXEC_available_telecommands(const char *args_str, TCMD_TelecommandCh
+    {
+        snprintf(response_output_buf, response_output_buf_len, "Could not set config var : %s to %lu\n", config_var_name, (uint32_t)config_var_new_value);
+        return 0;
+    }
+
+    snprintf(response_output_buf, response_output_buf_len, "Set config var: %s to %lu with width: %d \n", res->variable_name, *res->num , res->width_bytes);
+    return 0;
+}
+
+uint8_t TCMDEXEC_set_str_config_variable(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
+                                         char *response_output_buf, uint16_t response_output_buf_len)
+{
+    
+    char config_var_name[50];
+    memset(config_var_name, 0, 50);
+    const uint8_t parse_result = TCMD_extract_string_arg((const char *)args_str, 0,config_var_name, 50);
+    if (parse_result > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not parse index 0 for: ", args_str);
+        return 0;
+    }
+
+    char config_var_new_value[50];
+    memset(config_var_new_value, 0, 50);
+    const uint8_t parse_result2 = TCMD_extract_string_arg((const char *)args_str , 1, config_var_new_value, 50);
+    if (parse_result2 > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not parse index 1 for: ", args_str);
+        return 0;
+    }
+
+    CONFIG_string_config_entry_t *res;
+    const uint8_t found_int = CONFIG_find_string_variable_by_name((const char *)config_var_name, strlen((char *)config_var_name), &res);
+    if (found_int > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not find", config_var_name);
+        return 0;
+    }
+
+    char original_value[strlen(res->variable_pointer)];
+    strncpy(original_value, res->variable_pointer, res->max_length);
+    const uint8_t set_result = CONFIG_assign_str_new_value(&res->variable_pointer, config_var_new_value, res->max_length);
+    if (set_result > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "Could not set config var: %s because new value is too long\n", config_var_name);
+        return 0;
+    }
+    
+    snprintf(response_output_buf, response_output_buf_len, "Set config var: %s from \"%s\" to \"%s\" \n", config_var_name, original_value, res->variable_pointer);
+    return 0;
+}
+
+uint8_t TCMDEXEC_get_integer_configuration_variable(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
+                                                    char *response_output_buf, uint16_t response_output_buf_len)
+{
+    CONFIG_integer_config_entry_t *res;
+    const uint8_t found_int = CONFIG_find_integer_variable_by_name((const char *)args_str, strlen((char *)args_str), &res);
+    if (found_int > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not find", args_str);
+        return 0;
+    }
+
+    char json_buf[200];
+    memset(json_buf, 0, 200);
+    CONFIG_num_var_to_json(res, json_buf, 200);
```

Comment:
Replace 200 with `sizeof(json_buf)` on this line to indicate intention and reduce risk in case that buffer size is ever changed.

Any reason not to just write it out to `response_output_buf` right away?